internal/kafka/internal/clusters/standalone_provider.go

```diff
@@ -558,3 +558,8 @@ func (s *StandaloneProvider) CreateMachinePool(request *types.MachinePoolRequest
 	// TODO implement
 	return nil, nil
 }
+
+// noop method, it will always return an empty list as a standalone provider does not have any resource quotas
+func (s *StandaloneProvider) GetQuotaCost() ([]types.QuotaCost, error) {
+	return nil, nil
```

Comment:
Can we return an empty slice instead of nil?

Reply:
I think it's fine to return nil here? i think it's preferred in golang (see [here](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)). I believe they behave the same way, the built in functions for slices are still valid.

Reply:
We could do this if it makes it clearer?

```suggestion
        var quotaCostList []types.QuotaCost
	return quotaCostList, nil
```

Reply:
The description of the method indicates that an empty list is always returned. The description should match the behavior, whatever is the decided outcome.
Isn't the name of the method also a little bit confusing? It is named `GetQuotaCost`, singular, but a list of QuotaCost is returned. Shouldn't it be named `GetQuotaCosts` then?

Reply:
> I think it's fine to return nil here? i think it's preferred in golang (see [here](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)). I believe they behave the same way, the built in functions for slices are still valid.

Thanks.

Reply:
`GetQuotaCosts` sounds better, thanks @miguelsorianod. I'll update the name to that.

Reply:
updated it there, please have another look @machi1990 @miguelsorianod